src/components/scenes/RequestScene.js

```diff
@@ -321,11 +319,11 @@ export class RequestComponent extends React.Component<Props, State> {
                 </View>
               </InputAccessoryView>
             ) : null}
-            <View style={styles.qrContainer} onLayout={this.handleQrCodeLayout}>
-              {this.state.qrCodeContainerHeight < theme.rem(1) ? null : (
-                <QrCode data={this.state.encodedURI} size={this.state.qrCodeContainerHeight - theme.rem(1)} />
-              )}
-            </View>
+            <TouchableWithoutFeedback onPress={this.handleQrCodePress}>
+              <View style={styles.qrContainer}>
+                <LayoutDependentQrCode data={this.state.encodedURI} />
```

Comment:
`<LayoutDependentQrCode data={this.state.encodedURI} />` move it to const and use it twice